tests/face_test.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+import rhino3dm
+from pathlib import Path
+from honeybee.face import Face
+from honeybee_3dm.face import to_face
+
+
+def test_room():
+    path = str((Path.cwd()).joinpath('tests', 'test.3dm'))
```

Comment:
since we always run the tests from the root folder you can use relative path and keep it simple:

https://github.com/ladybug-tools/honeybee-energy/blob/95a436c0f6757b7ab8be72655d801fc936fca47e/tests/simulation_sizing_test.py#L25

I also suggest to put the 3dm file in a subfolder like `assets`.

Reply:
Complied